packages/frontend-app/src/app/components/Card/index.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { Card } from "./index";
```

Comment:
Please use this https://storybook.js.org/docs/react/writing-stories/decorators to make the background of the story black so the card can be visible